umake/frameworks/baseinstaller.py

```diff
@@ -396,7 +398,8 @@ def decompress_and_install_done(self, result):
             UI.return_main_screen(status_code=1)
 
         self.post_install()
-
+        if self.desktop_filename:
```

Comment:
Something like this?

@property
    def exec_link_name(self):
        if self.desktop_filename:
            return self.desktop_filename.split('.')[0]
        return None